x/concentrated-liquidity/math_test.go

```diff
@@ -6,17 +6,87 @@ import (
 	cl "github.com/osmosis-labs/osmosis/v12/x/concentrated-liquidity"
 )
 
-func (s *KeeperTestSuite) TestGetLiquidityFromAmounts() {
-	currentSqrtP, err := sdk.NewDecFromStr("70.710678")
-	sqrtPLow, err := sdk.NewDecFromStr("67.082039")
-	sqrtPHigh, err := sdk.NewDecFromStr("74.161984")
-	s.Require().NoError(err)
+func (suite *KeeperTestSuite) TestGetLiquidityFromAmounts() {
+	testCases := []struct {
+		name              string
+		currentSqrtP      sdk.Dec
+		sqrtPHigh         sdk.Dec
+		sqrtPLow          sdk.Dec
+		amount0Desired    sdk.Int
+		amount1Desired    sdk.Int
+		expectedLiquidity string
+	}{
+		{
+			"happy path",
+			sdk.NewDecWithPrec(70710678, 6),
+			sdk.NewDecWithPrec(74161984, 6),
+			sdk.NewDecWithPrec(67082039, 6),
+			sdk.NewInt(1),
+			sdk.NewInt(5000),
+			"1377.927096082029653542",
+		},
+	}
 
-	amount0Desired := sdk.NewInt(1)
-	amount1Desired := sdk.NewInt(5000)
+	for _, tc := range testCases {
+		tc := tc
 
-	s.SetupTest()
+		suite.Run(tc.name, func() {
+			liquidity := cl.GetLiquidityFromAmounts(tc.currentSqrtP, tc.sqrtPLow, tc.sqrtPHigh, tc.amount0Desired, tc.amount1Desired)
+			suite.Require().Equal(tc.expectedLiquidity, liquidity.String())
+		})
```

Comment:
can we add a check to indicate that this is same as result from `Liquidity1`, which is the min value?